lib/src/DateTimePicker/DateTimePicker.tsx

```diff
@@ -30,13 +30,15 @@ export type DateTimePickerProps = WithDateInputProps &
   WithViewsProps<'year' | 'date' | 'month' | 'hours' | 'minutes'>;
 
 function useDefaultProps({
-  ampm = false,
+  ampm,
```

Comment:
Is the plan to remove this option?

Reply:
No, because it is still required for locales where both variants are in use. So it is possible to force 12/24h format

Reply:
I thought about make something like `clockCycle="12h"|"24h"` or just `clockMode`. For the public api, but left `boolean` under the hood